GVFS/GVFS.Platform.Mac/ProjFSKext.cs

```diff
@@ -73,5 +69,40 @@ public bool TryPrepareFolderForCallbacks(string folderPath, out string error, ou
 
             return true;
         }
+
+        public bool TryLoad(ITracer tracer, string enlistmentRoot, out string errorMessage)
+        {
+            EventMetadata metadata = new EventMetadata();
+            ProcessResult loadKext = ProcessHelper.Run("sudo", "/sbin/kextload -b " + DriverName);
+            if (loadKext.ExitCode == LoadKext_ExitCode_Success)
+            {
+                tracer.RelatedWarning(metadata, $"{DriverName} was successfully loaded.", Keywords.Telemetry);

```

Comment:
What was the reason for making this a Warning?

Reply:
I wanted this to be a little stronger than Informational because this would mean that the kext for some reason wasn't auto loaded by the OS.  But if this is too strong I can make it Informational and add some text to the message so that we know this usually shouldn't happen.  Maybe I should add the text in any case?

Reply:
>But if this is too strong I can make it Informational

Thanks for the explanation, I think it's OK to leave it as Warning.  

> Maybe I should add the text in any case?

Adding a bit more text for better context sounds good to me.